packages/typespec-test/test/openai_generic/generated/typespec-ts/src/models/models.ts

```diff
@@ -501,7 +501,7 @@ export interface Embedding {
   object: "embedding";
   /**
    * The embedding vector, which is a list of floats. The length of vector depends on the model as\
-   *    * listed in the [embedding guide](/docs/guides/embeddings).
+   * listed in the [embedding guide](/docs/guides/embeddings).
```

Comment:
The change should be expected and fixed here: https://github.com/microsoft/typespec/issues/3646.


which means the nightly ci would be expected to be failed.